src/databases/models/User.js

```diff
@@ -46,6 +46,10 @@ module.exports = (sequelize, DataTypes) => {
       country_code: {type: DataTypes.STRING, allowNull: false},
       username: {type: DataTypes.STRING, allowNull: false, unique: true},
       real_name: {type: DataTypes.STRING, allowNull: true},
+      encrypted: {
+        type: DataTypes.STRING,
+        allowNull: true
+      },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `encrypted` field is fine, but it's not clear what kind of data this field will hold. If it's going to store encrypted sensitive user information, consider adding a comment to clarify its purpose. Also, ensure that you're following all necessary security measures when dealing with encrypted data.
```diff
+      // This field stores encrypted sensitive user information
       encrypted: {
         type: DataTypes.STRING,
         allowNull: true
       },
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->